packages/docs/docs/concepts/progressive-loading/requirements.md

```diff
@@ -0,0 +1,301 @@
+---
+id: requirements
+---
+
+# Requirements and Configuration for Progressive Loading
+
+Fast initial display of images requires some method of being able to retrieve
+just part of an image or volume that can be rendered as a complete but lossy image.
+For example, a thumbnail image could be rendered full size, or images in a volume
+could be interpolated to produce an alternate image. These images are retrieved
+first for fast initial display, followed by retrieving a full resolution image,
+thus producing a progressively better display as more data is loaded.
+
+The DICOM standards committee is currently adding support in DICOM for
+a new encoding method, High Throughput JPEG 2000 `htj2k`. This encoding
+method can be configured to allow for progressive decoding of images.
+That is, if the first N bytes of the image encoding are available they can be
+decoded to a lower resolution or lossy image. The configuration that allows this
+is the HTJ2K Progressive Resolution (HTJ2K RPCL).
```

Comment:
some link to the document is appreaciate here, where people should track htj2k in the standard

Reply:
what is RPCL stand for?

Reply:
It isn't in the standard - I'm not sure where to put the link to right now.  Will update it when it arrives.